File: 0027_Remove_Element/Remove_Element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int pos = 0;
        
        for (int x : nums) {      // x is values here
            if (x != val) {
                nums[pos++] = x;
            }
        }
        
        return pos;
    }
};

File: 0027_Remove_Element/Remove_Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        pos = 0
        
        for x in nums:
            if (x != val):
                nums[pos] = x
                pos += 1
        # Val bade jesob value asbe segulo ke allocate korbe r next tai ready hoye bose thakbe, val esegele skip kore jabe
        return pos # Oitai abar return kore debe